t/test.lua

```diff
@@ -196,6 +196,13 @@ function issue_120(request)
   return request
 end
 
+function skipping_test(...)
+  for k, v in pairs({...}) do
```

Comment:
AFAIU this `print()` doesn't affect anything. Am I right?

Reply:
`print()` calls `__tostring`, so we have to see this methods.
On the other hand, it is this is just a test. Even if it affects something, the main thing is that the test produces the expected result. Or am I not aware of something?

Reply:
After discussion with Roman, it was decided that the print is not needed.